ArduSub/ArduSub.cpp

```diff
@@ -344,4 +346,15 @@ void Sub::update_altitude()
     }
 }
 
+void Sub::update_strobe_light()
+{
+    if(RC_Channels::get_radio_in(11) < 1500) {
+        return;
+    }
+
+    static bool strobe_state;
```

Comment:
Ardupilot guidelines are now for no in-line static variables. Move this to Sub.h please.